api/server/router/container/container_routes.go

```diff
@@ -643,6 +643,20 @@ func (c *containerRouter) postContainersCreate(ctx context.Context, w http.Respo
 		}
 	}
 
+	if versions.LessThan(version, "1.48") {
```

Comment:
Oh we also discussed not gating this on API.
@tianon @thaJeztah Did we come to a conclusion on this?

Reply:
I'm +1(00) on dropping the API version limitations (with thanks and apologies to vbatts for having implemented it successfully), but I think Seb still had some reservations unless I managed to be convincing enough. :bow: :heart: